README.md

```diff
@@ -64,6 +64,8 @@ func main() {
 }
 ```
 
+If a value in the connection string contains a `;` you need to escape it with `\;`. This ensures that the driver can parse the connection string as expected.
```

Comment:
:+1: 

I wanted to add a paragraph like "In this case, please ensure to escape all values properly."
But your proposal is even better and more concrete. 
Thank you!